src/api/Ethereum.ts

```diff
@@ -37,6 +37,14 @@ export type Tx = {
     }>;
     truncated: boolean;
   };
+  erc721TransferEvents?: {
+    list: Array<{
+      contract: string;
+      from: string;
+      to: string;
+      token_id: BigNumber;
```

Comment:
Typing here is wrong and should be `number` or `string` since `Tx` are casted directly from the `fetchAllTransactions` so every types for `Tx` should be a serialized type. 
**But** since this issue is true in the whole `Tx` type, I kept up with the flow.